index.html

```diff
@@ -19,6 +19,55 @@
     <!--HEADER SECTION -->
 
     <div class="header-container">
+        
+        <!-- <div class="details-modal">
+            <div class="sub-modal">
+                <button class="close-pop">
+                    <i class="fa-solid fa-xmark"></i>
+                </button>
+            <div>
+                <h3 class="pop-heading">
+                Tonic
+                </h3>
+            </div>
+            <div class="list-items">
+                <h4>CANOPY</h4>
+                <ul class="ul-list">
+                    <li>Back End Dev</li>
+                </ul>
+                <ul>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="pop-img">
+                <img src="images/Snapshoot-Portfolio-5.png" alt="">
+            </div>
+            <div class="desk-pop-img">
+                <img src="images-desk/Snapshoot-Portfolio-popup.png" alt="">
+            </div>
+
+            <div class="modal-para">
+            <p class="pop-message">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem perspiciatis, harum hic culpa libero, accusamus doloribus eius explicabo, modi quae aspernatur illo ad beatae assumenda? Ipsam minus beatae ratione aliquid ullam quaerat sed molestiae sit eum pariatur? Amet totam, voluptas sapiente voluptate, consectetur laudantium delectus molestiae maiores officia explicabo soluta!
+            </p>
+            </div>
+            <div>
+                <ul class="pop-tech">
+                    <li><button type="button" class="btn-1">html</button></li>
+                    <li><button type="button" class="btn-2">css</button></li>
+                    <li><button type="button" class="btn-3">javaScript</button></li>
+                </ul>
+            </div>
+            <hr>
+            <div>
+                <ul class="pop-button">
+                    <li><button type="button" class="pop-btn">See live <img src="images/Icon1.svg" alt=""></button></li>
+                    <li><button type="button" class="pop-btn">See Source <img src="images/Vector2.svg" alt=""></button></li>
+                </ul>
+            </div>
+            </div>
+        </div>
+    -->
```

Comment:
- [ ] It's considered bad practice to add commented-out pieces of code in your Files as it clutters the info. Kindly remove all commented out pieces of code